app/views.py

```diff
@@ -37,6 +41,37 @@ def upload():
     return flask.render_template('upload.html', form = form)
 
 
+@app.route("/about")
+def about():
+    """
+    Define the about route
+    """
+    nb_pdbs = db.session.query(func.count(model.PDBFile.id)).scalar()
+    print(nb_pdbs)
+
+    num_P = 0
+    for annotation in model.Annotation.query.all():
+        num_P += annotation.result.count('P')
+    print (num_P)
+
+    mean_resol = 0
```

Comment:
http://stackoverflow.com/questions/7143235/how-to-use-avg-and-sum-in-sqlalchemy-query